lib/upload.js

```diff
@@ -23,7 +23,10 @@ const defaultOptions = {
   addRequestId: false,
   onBeforeRequest: null,
   onAfterResponse: null,
-  onShouldRetry: null,
+  onShouldRetry: (err) => {
```

Comment:
Please don't use an anonymous function here but put it in a function called `defaultOnShouldRetry`. This keep the definition of `defaultOptions` cleaner.

Reply:
done